app/build.gradle.kts

```diff
@@ -3,6 +3,7 @@ import java.io.FileInputStream
 import java.util.Properties
 
 plugins {
+    kotlin("plugin.serialization") version "1.8.10"
```

Comment:
Warning: Is this a hardcoded version in `build.gradle.kts` ? 👀
Also, you didn't add `jetbrainsKotlinxSerializationJson ` and `gson` package to the dependencies. Is there a reason ?

Reply:
Sorry for this. I forgot to review the changes of the `build.gradle` all things were done as temporary I commited the fixes for these issues.